.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
 | 🚥 Fix RM-XXX |
```

Comment:
Thoughts on renaming this? I've been thinking about changing this for a while now

```suggestion
| 🚥 Fixes RM-XXX |
```

Reply:
let's do it

Reply:
since this is applying on all OSS repos, thoughts on removing the `RM-` prefix?

Reply:
100%, having that in those repos has always bothered me